src/vm/win32threadpool.cpp

```diff
@@ -1815,8 +1815,8 @@ Thread* ThreadpoolMgr::CreateUnimpersonatedThread(LPTHREAD_START_ROUTINE lpStart
         // CreateNewThread takes care of reverting any impersonation - so dont do anything here.
         bOK = pThread->CreateNewThread(0,               // default stack size
                                        lpStartAddress,
-                                       lpArgs           //arguments
-                                       );
+                                       lpArgs,           //arguments
+                                       W("ThreadPool"));
```

Comment:
From a descriptive name perspective that makes sense. But the longer the name gets the more screen real estate is wasted when one tries to view the data along with many other columns. I would therefore use as short names as possible. 
An example is here: https://randomascii.wordpress.com/2015/10/26/thread-naming-in-windows-time-for-something-better/
I can change it if you wish.